### DISCUSSION TITLE
#[Modelable] break session flashed vars

### DISCUSSION THREAD

**Discussion by AunCly** [livewire/livewire#7022] [7. Bugs (archived)]
### Livewire version

3.0.6

### Which PHP version are you using?

PHP 8.1

### Repository URL

https://github.com/AunCly/livewire_modelable_issue

### Steps To Reproduce

Hello,

I have a bit tricky issue but i will try to do my best to explain it. It seems that #[Modelable] tag break the flashed session vars, but i didn't find why.

1 - Make a layout with a condition to display your session flashed notification 
Like this : https://github.com/AunCly/livewire_modelable_issue/blob/main/resources/views/layout.blade.php

2 - Create a parent component with properties and an action who flash session vars and redirect, and in the view add a nested livewire component and give it a wire:model
Component : https://github.com/AunCly/livewire_modelable_issue/blob/main/app/Livewire/Contact.php
View : https://github.com/AunCly/livewire_modelable_issue/blob/main/resources/views/livewire/contact.blade.php

3 - Create a nested component with #[Modelable] attribute tag
Component : https://github.com/AunCly/livewire_modelable_issue/blob/main/app/Livewire/CustomInput.php
View : https://github.com/AunCly/livewire_modelable_issue/blob/main/resources/views/livewire/custom-input.blade.php

With #[Modelable] the notification won't display, but if we remove the #[Modelable] tags all work normally.

The session vars are a bit différent if we look at the telescope session tab : 
With #[Modelable] tag : 
![Capture d’écran 2023-10-09 à 10 28 19](https://github.com/livewire/livewire/assets/6899316/cd5fe2c0-c9f4-48e5-949b-a75d025cf369)

And without it : 
![Capture d’écran 2023-10-09 à 09 56 49](https://github.com/livewire/livewire/assets/6899316/9e124097-3c2b-4b2b-9a53-37bb5560b0cb)

Hope it's clear, 

Thanks you :)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by desarrollolandatelc** [comment]
Have you found the solution? I have the same issue, but I haven't been able to find the solution

**Reply by AunCly** [reply]
Nope, i'm waiting for solution, so i'm not using in in the meantime :/
